lib/Dialect/SystemC/SystemCOps.cpp

```diff
@@ -173,6 +186,51 @@ LogicalResult SCModuleOp::verify() {
   return success();
 }
 
+LogicalResult SCModuleOp::verifyRegions() {
+  DenseSet<StringRef> memberNames;
+  DenseSet<StringRef> localNames;
```

Comment:
To facilitate better error reporting where the list of all conflicting ops are printed, these could be changed to use `llvm::EquivalenceClasses` (a disjoint set) or `DenseMap<StringRef, SmallVector<Operation *>>`. This is likely overkill and it is sufficient to just print out the ops that conflict as soon as they are found.